theme/src/components/frame.js

```diff
@@ -6,7 +6,7 @@ import Measure from 'react-measure'
 function Frame({children}) {
   const [height, setHeight] = React.useState('auto')
   return (
-    <FrameComponent style={{width: '100%', border: 0, height}}>
+    <FrameComponent style={{width: '100%', border: 0, border-radius: '6px', height}}>
```

Comment:
React expects style properties to be camelCased:

```suggestion
    <FrameComponent style={{width: '100%', border: 0, borderRadius: 6, height}}>
```